### PR TITLE
move v2 code to its own file

### PR DESCRIPTION
**Pull request by BlowaterNostr**

### CODE REVIEW

File: nostr.ts [hunk 1]
```diff
@@ -221,35 +181,16 @@ export class DecryptionFailure extends Error {
...
-    return hexEncode(sha256(buf));
+    return encodeHex(sha256(buf));
 }
 
 function eventCommitment(event: UnsignedNostrEvent): string {
     const { pubkey, created_at, kind, tags, content } = event;
     return JSON.stringify([0, pubkey, created_at, kind, tags, content]);
 }
 
-function hexEncode(buf: Uint8Array) {
```
Comment (BlowaterNostr) [inline nostr.ts:232]: use deno std lib to have cleaner code

File: nostr.ts [hunk 2]
```diff
@@ -376,3 +302,4 @@ export * from "./websocket.ts";
 export * from "./event.ts";
 export * from "./key.ts";
 export * from "./relay.interface.ts";
+export * as v2 from "./v2.ts";
```
Comment (bob2402) [inline nostr.ts:305]: After releasing the new version, when depending on updating this version, we need to modify the imported v2.

**Review by bob2402 — APPROVED** [review]
LGTM

File: nostr.ts (hunk 2 above)
Comment (BlowaterNostr) [reply nostr.ts:297]: yes

**Pull request merged**